MarsOnboarding24/Features/Language.feature

```diff
@@ -1 +1,53 @@
-﻿
\ No newline at end of file
+﻿Feature: Language
+
+1. Creating a language record
+2. Editing a language record
+3. Deleting a language record
+4. Adding more language records than the minimum permitted
+5. Creating a language record using special characters
+6. Creating a language record with null data
+7. Creating a language record with existing data
+
+Scenario: A. Creating a language record
+	Given user logs into the portal
+	And user navigates to the Languages tab
+	When user creates a new Language record
+	Then system should save the new Language record
+
+Scenario: B. Editing a language record
+	Given user logs into the portal
+	And user navigates to the Languages tab
```

Comment:
Please use tables  and pass values to your scenarios for example look at the below scenario 
This way you will get exposure to use them  and will know how to pass test data to the tests


Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

See  https://cucumber.io/docs/gherkin/reference/

Reply:
make sure you apply this to all feature file